safe_core/src/client/mock/tests.rs

```diff
@@ -1241,6 +1359,7 @@ fn auth_actions_from_app() {
     send_req_expect_ok!(
         &mut connection_manager,
         &client_safe_key,
+        token_signed_by_client.clone(),
```

Comment:
Sending token as a client

Reply:
this is part of `auth_actions_from_app` test suite. so I can only assume that actually this should be using the `app_conn_manager` ?

Reply:
Not exactly, we upload the MD initially as the client. And then proceed with the tests.